chart/gatekeeper-operator/templates/gatekeeper.yaml

```diff
@@ -511,3 +526,23 @@ webhooks:
     - '*'
   sideEffects: None
   timeoutSeconds: 5
+- clientConfig:
+    caBundle: Cg==
+    service:
+      name: gatekeeper-webhook-service
+      namespace: gatekeeper-system
+      path: /v1/admitlabel
+  failurePolicy: Fail
```

Comment:
I'm debating with myself if this is a good idea. 
- On the one hand, we don't want labels to be applies on namespaces if the gk service is down. But on the other hand, I could see this blocking updates on namespaces in the cluster that aren't even related to gk if the gk service is down. Without HA this can happen.

Reply:
This is true, but we assume GK being down is a recoverable state, as we aren't blocking Pod creation.

Basically this weighs the likelihood of people needing to create a namespace right this moment against the severity of bypassing policy checks.

Maybe we could add a DR section to the doc that tells people to delete the webhook if it's causing issues?

With HA this can happen too, it's just less likely to happen.

Reply:
> add a DR section to the doc that tells people to delete the webhook if it's causing issues

+1

I'm curious what other people think about this.

Reply:
Ya, though I suspect we wont get much traffic on this thread. I'm going to operate on the assumption that this is the way to go unless I hear differently.

Reply:
Added DR section to README.